### ISSUE TITLE
[CAIA A11y Research - PILOT]: Veteran-Facing Forms, VA Form 21-4138 (July 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#88931]
> [!IMPORTANT]
> - **TICKET STATUS:** `COMPLETED`
> - **PILOT DATE:**  7/18/2024 at 12:30 p.m. ET (Confirmed @EvanAtCoforma)
> - **TYPES OF ASSISTIVE TECH:** 
> - **PROTOTYPE**: [Prototype in Staging](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/introduction)

# CAIA A11y Support
This is a child ticket of the research ticket  #86767 - created to track work, completed from this pilot session. 

> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

@EvanAtCoforma 

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name: Veteran Facing Forms
- OCTO-DE product owner: Matthew Dingee
- Product manager: Tricia Baker
- Designer: Rachel Shearer
- FE engineer: Jacob Penner, Robert Hasselle
- Product/team Slack channel: #veteran-facing-forms

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
**Date:** Thursday, July 18 from 12:30 – 2:00pm `CONFIRMED`

#### Pilot Resources
- Internal CAIA Notes Doc [WIP](https://docs.google.com/document/d/1dJJruCuCKuPjb9kR02AKUiDcWg67byE_wFHxqJG8lyk/edit?usp=sharing)
- [Prototype in Staging](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/introduction)
- GitHub Resources
    - [Research plan](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/research/2024-03-reduce-4138-submissions/Research%20Plan.md)
    - [Research conversation guide](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/research/2024-03-reduce-4138-submissions/Conversation%20guide.md)
    - [Assistive Technology Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md#assistive-technology-pilot-guide)

</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Research Goals

- Evaluate the usability of a pre-form subtask that asks users to identify their statement type and explore relevant alternatives before completing a 21-4138.
- Evaluate the effectiveness of presenting a subtask to users in the flow of accessing the form.

### Research Questions

**Responses, based on the pilot by @EvanAtCoforma**: 

- How effective is a dynamic pre-form subtask? 
    - [x] Effective
    - [ ] Partially Effective
    - [ ] Not Effective
- Do participants choose paths that correlate to their described goals?
    - [x] Yes
    - [ ] No
    - [ ] Partially 
- How are AT users and users with cognitive impairments affected by the interactive pre-form sub-tasks?
    - [ ] Affected
    - [x] Partially Affected
    - [ ] Not Affected
- How do users familiar with the 4138 respond to the list of alternatives?
    - [x] Likely to like
    - [ ] Likely to dislike
    - [ ] Likely to neither dislike or like

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11y:** @EvanAtCoforma
- [x] **Share link to [Assistive Tech Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with team, if they haven't reviewed it yet**
- [x] **Determine date and time for pilot:**
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled
- [x] **Review research** [conversation guide](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/research/2024-03-reduce-4138-submissions/Conversation%20guide.md) and provide feedback in comments.
- [x] [Request AT device and tech to be used in sessions](LINK)
- [x] [Receive AT device and tech from Team in research ticket](LINK) to be used in sessions to prep for pilot
- [x] Review Goals in Research Plan (Evan and Sara) - added to ticket
- [x] [Plan/create WIP prep doc](LINK)
- [x] **Review prototype** - [Prototype in Staging](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/introduction)
- [x] **Run pilot** with VFS Researcher
- [ ] **Google Doc:** [Pilot Prep Doc for Evan](https://docs.google.com/document/d/1dJJruCuCKuPjb9kR02AKUiDcWg67byE_wFHxqJG8lyk/edit?usp=sharing)
```

**Comment by sara-amanda** [comment]
# CAIA Pilot Feedback

> [!NOTE]
> **Research Goals:** Evaluate the usability of a pre-form subtask that asks users to identify their statement type and explore relevant alternatives before completing a 21-4138. Evaluate the effectiveness of presenting a subtask to users in the flow of accessing the form.

## Assistive Technology
- **Type**: Mac
- **Device**: Laptop
- **Assistive Technology:** Voiceover
- **Pilot**: @EvanAtCoforma 
- **Co-Pilot:** NA

## Pilot Takeaways
- **Were there any accessibility issues of note?**
    - [x] Aside from a table not reading its’ label, there were no accessibility issues. 
    - [x] The team already fixed this.
-  **Are the research goals met? If no, list those not met.**
    - [x] Research goals seemed to have been met, from an accessibility perspective, though this would be more a question for the VA team that built the prototype. 

### Pilot Notes

- Going through as someone who isn’t familiar with the different types of VA forms, it took a little time to understand that there was an alternative form that was better geared for the task at hand rather than the 4138.
- Once I understood that there was a better alternative, the process of getting to and filling out the form was simple.
- There was what I considered a duplicate screen within the prototype
    - There was an initial screen that gave an explanation of what to expect and what was needed, then a start form button
    - The next screen seemed to give the same information, which was confusing since I assumed I’d go into the form according to the language of the previous screen
- There was a table that didn’t announce its’ label
    - This issue has already been fixed, as the team added in the v3 version of the table which reads correctly 

### Prototype
[Prototype in Staging](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/introduction)
### GitHub Resources
- [Research plan](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/research/2024-03-reduce-4138-submissions/Research%20Plan.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/research/2024-03-reduce-4138-submissions/Conversation%20guide.md)
- [Assistive Technology Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md#assistive-technology-pilot-guide)
